src/coreComponents/physicsSolvers/CMakeLists.txt

```diff
@@ -133,6 +124,34 @@ if ( ENABLE_CUDA )
   set( dependencyList ${dependencyList} cuda )
 endif()
 
+if( ENABLE_PVTPackage )
+    list( APPEND physicsSolvers_headers
+         fluidFlow/CompositionalMultiphaseBase.hpp
+         fluidFlow/CompositionalMultiphaseFVM.hpp
+         fluidFlow/CompositionalMultiphaseHybridFVM.hpp
+         fluidFlow/CompositionalMultiphaseBaseKernels.hpp
+         fluidFlow/CompositionalMultiphaseFVMKernels.hpp
+         fluidFlow/CompositionalMultiphaseHybridFVMKernels.hpp
+         fluidFlow/wells/CompositionalMultiphaseWell.hpp
+         fluidFlow/wells/CompositionalMultiphaseWellKernels.hpp
+         multiphysics/MultiphasePoroelasticSolver.hpp
+         multiphysics/CompositionalMultiphaseReservoir.hpp
+         multiphysics/MultiphasePoroelasticKernel.hpp
+         )
+
+    list( APPEND physicsSolvers_sources
+         fluidFlow/CompositionalMultiphaseBase.cpp
+         fluidFlow/CompositionalMultiphaseFVM.cpp
+         fluidFlow/CompositionalMultiphaseHybridFVM.cpp
+         fluidFlow/CompositionalMultiphaseBaseKernels.cpp
+         fluidFlow/CompositionalMultiphaseFVMKernels.cpp
+         fluidFlow/CompositionalMultiphaseHybridFVMKernels.cpp
+         multiphysics/CompositionalMultiphaseReservoir.cpp
+         multiphysics/MultiphasePoroelasticSolver.cpp
+         fluidFlow/wells/CompositionalMultiphaseWell.cpp
+         )
+endif()
```

Comment:
This is not correct anymore, we have multiphase constitutive models that run natively in GEOSX, without `PVTPackage`. Perhaps constitutive dispatch needs to be fixed to avoid `PVTPackage`-dependent models when it's not enabled. We can do it in a follow up.

Reply:
made an issue for this.